web/src/settings_preferences.ts

```diff
@@ -255,7 +280,7 @@ export function set_up(settings_panel: SettingsPanel): void {
     // element is changed.
     $container.on(
         "change",
-        "input[type=checkbox], .information-density-settings input[type=text], select",
+        "input[type=checkbox], .information-density-settings input[type=text], select, .setting_color_scheme",
```

Comment:
Is there a reason to include the event handling of the colro_scheme in this block of code. I don't see much relevance except for the common `$status_element`, and there is already too many things happening this event handler.

Reply:
I suggest you rather have a new event handler like:
```ts
$container.find(".setting_color_scheme").on("change", ...)
```
and include the code inside `change_user_theme` directly there, since I also don't see any other calls to the `change_user_theme` being needed in the current state of settings.

Reply:
I included the event handling in the particular block of code because there is a comment in the `settings_preferences.ts` file that states:
```
  // Common handler for sending requests to the server when an input
  // element is changed.
```

So, I thought it would be better to include the event handling for the `color_scheme` in the same block.
However, I agree that it would be better to create a new event handler for the `color_scheme`. I have updated the changes. Thank you.